src/odemis/gui/cont/acquisition.py

```diff
@@ -515,6 +530,11 @@ def _onROA(self, roi):
         self.btn_acquire.Enable(roi != UNDEFINED_ROI)
         self.update_acquisition_time()  # to update the message
 
+    @call_in_wx_main
+    def on_preparation(self, is_preparing):
+        self.btn_acquire.Enable(not is_preparing)
```

Comment:
Enabling the acquisition button needs to be global (iow, also does what onROA() does).
Create a new _check_acquire_button()
=> It checks if roi != UNDEFINED _and_ not is_preparing